BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -49,19 +49,56 @@ def signing_keys():
 
         # CONTAINER_COMP_KEY_ID is used for signing container components
         # One can add multiple component keys as needed.
-        ("CONTAINER_COMP_KEY_ID",      "ContainerCompTestKey_Priv"),
+        ("CONTAINER_COMP_KEY_ID",       "ContainerCompTestKey_Priv"),
+        ])
+
+    return signing_keys_list
+
+def signing_pub_keys():
+    signing_keys_list = []
+    signing_keys_list.append ([
+      # Key ID                          | Key File Name start |
+      # ===========================================================
+        # OS1_PUBLIC_KEY_ID is used for referencing Boot OS public keys
+        ("OS1_PUBLIC_KEY_ID",           "OS1_TestKey_Pub"),
+
         ])
 
     return signing_keys_list
 
 # Generate RSA Key based on required key size
-def GenerateRsaKeys (openssl_path, key_dir, key_size):
-    key_list = signing_keys()
+def GenerateRsaPrivKeys (openssl_path, key_dir, key_size):
+    key_list = signing_priv_keys()
     for item in key_list:
         for Key_name, Key_file_name in item:
             cmd = '%s genrsa -F4 -out %s/%s_RSA%s.pem %s' % (openssl_path, key_dir, Key_file_name, key_size, key_size)
             run_process (cmd.split())
 
+    # Generate test public keys
+    GenerateRsaPubKeys (openssl_path, key_dir, key_size)
+    return
+
+# Generate RSA Public Keys based on required key size
+# User is required to replace the public keys with their original keys.
+def GenerateRsaPubKeys (openssl_path, key_dir, key_size):

```

Comment:
Generating dummy/test public keys for OS image. User/customer needs to replace with appropriate keys.